src/components/LicenseRemindModal/index.jsx

```diff
@@ -78,6 +78,9 @@ class LicenseRemindModal extends React.Component {
       isBulkRemind,
       user,
       sendLicenseReminder,
+      fetchSubscriptionDetails,
+      fetchSubscriptionUsers,
+      searchQuery,
```

Comment:
These values are already in the `SubscriptionContext` that can be read from within any of the subscriptions component tree (`const value = useContext(SubscriptionContext)`, as opposed to needing to pass them around as props into various components 😄

Reply:
While ding it this way I am getting an error
Invalid hook call. Hooks can only be called inside of the body of a function component.
I think it is because we are trying to use it in class base component.

Reply:
Yeah, hooks can only be used in functional component. To use context in a class-based component: https://reactjs.org/docs/context.html#classcontexttype